myriad-scheduler/src/main/java/org/apache/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java

```diff
@@ -243,7 +243,7 @@ private boolean meetsConstraint(Offer offer, Constraint constraint) {
 
   private void checkResource(boolean fail, String resource) {
     if (fail) {
-      LOGGER.info("No " + resource + " resources present");
+      LOGGER.error("No " + resource + " resources present");
```

Comment:
Again not an error, it's possible to have an offer with 0.0 cpu resources, but with 2048 mem resources.  While it's not useful to the framework, it's not an error.  Debug or info.